File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.java
class Solution {
    public int[] replaceElements(int[] arr) {
        int n=arr.length;
        int [] ans= new int[n];
        for(int i=0; i<n; i++){
            int max = -1;
            int j=i+1;
            while(j<n){
                if(arr[j]>max){
                    max= Math.max(max, arr[j]);
                }
                j++;
            }
            ans[i]=max;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/counting-bits/submission-0.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int>ans(n+1); 
        ans[0]=0; 
        int i=1; 
        while(i<=n){ 
            ans[i]=ans[i/2]+i%2; 
            i++; 
        
        } return ans;
    }
};
